src/commands/rsync.ts

```diff
@@ -91,6 +92,7 @@ export const rsyncCommand = async () => {
 
 	try {
 		terminal.sendText(`pnpm jetpack rsync ${plugin} ${wpPath}`);
+		callTrackEvent('wpcom_test_test');
```

Comment:
Should we name this `jetpackvsc_command_exec` before merging?

Reply:
Yup that's a good name, done 👍